_posts/2022-01-14-december-rubygems-updates.md

```diff
@@ -0,0 +1,29 @@
+---
+title: December 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+# December 2021 Monthly Update
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Central](http://rubytogether.org/), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in December.
+
+## RubyGems News
+
+In December, we released new versions for RubyGems [3.3.0](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#330--2021-12-21), [3.3.1](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#331--2021-12-22), [3.3.2](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#332--2021-12-23), [3.3.3](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#333--2021-12-24), [3.3.4](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#334--2021-12-29) and Bundler [2.2.33](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2233-december-7-2021), [2.3.0](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#230-december-21-2021), [2.3.1](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#231-december-22-2021), [2.3.2](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#232-december-23-2021), [2.3.3](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#233-december-24-2021), [2.3.4](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#234-december-29-2021).
+
+We also worked on Bundler version switching, fixed a couple small regressions, and improved some resolution errors.
```

Comment:
For my side, I would change this to something like

> With this batch of releases, we finally shipped the Bundler version switching feature, which had been planned for some time now, and provided final versions to be included with new Ruby 3.1 release.

Reply:
thanks for the input. I'll update the post.

Reply:
Thank you as always @lauragift21!